viper/compile_lll.py

```diff
@@ -93,7 +93,9 @@ def compile_to_assembly(code, withargs=None, break_dest=None, height=0):
     # Repeat(memloc, start, rounds, body)
     elif code.value == 'repeat':
         o = []
-        loops = num_to_bytearray(code.args[2].value) or [2]
+        loops = num_to_bytearray(code.args[2].value)
+        if not loops:
+            raise Exception("Number of times repeated must be a constant nonzero positive integer: %r" % loops)
```

Comment:
Works for me, however as I see it - we need a way to test this. Perhaps using your new get_contract_from_lll fixture ;)